pom.xml

```diff
@@ -499,4 +506,17 @@
 			</plugin>
 		</plugins>
 	</build>
+
+	<distributionManagement>
```

Comment:
I am not sure this is needed with the right settings.xml.

Reply:
yeah this shouldn't be there. Please remove.

Reply:
We need it here. Please check with David Zhang